### PR TITLE
WIP: Add GDI08a

### PR DESCRIPTION
**Pull request by Muh-Muh**
Work in Progress: Feedback and help appreciated

The AI mostly is based on the global script for d2k. 

ToDo:
- [x] Replace map.png
- [ ] Prevent units from cluttering the base  (This sort of works, but is far from optimal)
- [ ] Balancing (less money for hard/normal?, different attack group sizes?)
- [x] Prevent newly built vehicles from stacking.
- [ ] Make infantry tab clickable

- [ ] Whitespace fixes :)

Nice to Have:
- [ ] AI Resource Management (right now I give the AI a constant amount of money, so it doesn't run out)
- [ ] AI rebuilds its base
- [ ] coordinated attacks (right now faster vehicles arrive first at the players base, making engagements less challenging and fun)
- [ ] Other?

### CODE REVIEW

File: mods/cnc/maps/gdi08a/gdi08a-AI.lua [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+NodInfantryTypes = { "e1", "e1", "e1","e3", "e3", "e4", "e4"}
+NodVehicleTypes = { "bggy", "bggy", "bike", "ltnk"}
+
+
+ActivateAI = function()
+	IdlingUnits[Nod] = {}
+	DefendAndRepairBase(Nod, NodBase, 1, AttackGroupSize[Difficulty])
+	DefendAndRepairBase(Nod, Samsites, 1, AttackGroupSize[Difficulty])
+
```
Comment (Smittytron) [inline mods/cnc/maps/gdi08a/gdi08a-AI.lua]: For the vehicle's stacking I'd go with `NodAirfield.RallyPoint = waypoint4.Location`

**Comment by Mailaender** [comment]
> Replace map.png (where do I get the appropriate image?)

I think you need to extract those with XCC mixer and then use an image editor to stich it together.

**Comment by Muh-Muh** [comment]
Updated.
As the long period of inactivity shows, I am strapped for time at the moment. This won't change for the foreseeable future so further updates might take a long time. Feel free to close this PR/Take it over if this gets in the way of things.

I also noticed that the infantry-tab is not highlighted and unclickable at the beginning of the mission. I guess it must be a wrong YAML value somewhere, but I can't figure it out.

**Comment by pchote** [comment]
It looks like there's still quite a way to go here, and as mentioned above not much chance of this being mergable any time soon.  I'll close this to help clear up the PR queue, but please reopen this when you are able to get back to it.

**Pull request closed without merging**